documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

```diff
@@ -52,6 +55,8 @@ To set up the Compute Module IO Board:
 
 Next, let's set up software on the host device.
 
+The recommended host device is a Raspberry Pi 4 or newer running Raspberry Pi OS 64-bit.
```

Comment:
```suggestion
TIP: For a host device, we recommend a Raspberry Pi 4 or newer running 64-bit Raspberry Pi OS.
```

Suggestion: convert this into a TIP admonition, since it isn't essential to the guide flow.